src/components/AddBoxForm.js

```diff
@@ -19,7 +19,7 @@ const AddBoxForm = ({ classes, onClose, onSubmit, products }) => (
         errors.product = "Select a product.";
       }
       if (!values.quantity) {
-        errors.quantity = "Enter the number of items.";
```

Comment:
This should probably be a separate commit. Not a problem for now - but for future reference!